.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "7"
+  - "8"
```

Comment:
Should this be updated as well?

https://github.com/brave/browser-laptop/blob/836d3916f8941deca1aac65ec47aea66eae62b12/package.json#L230-L232